test/BooleanFromString.ts

```diff
@@ -0,0 +1,11 @@
+import { BooleanFromString } from '../src/BooleanFromString'
+import { assertSuccess, assertFailure } from './helpers'
+
+describe('BooleanFromString', () => {
+  it('decode', () => {
+    const T = BooleanFromString
+    assertSuccess(T.decode('true'), true)
+    assertSuccess(T.decode('false'), false)
```

Comment:
You should split this test in two different tests, one for `'true'` and one for `'false'`.